algolia/analytics/utils.go

```diff
@@ -10,7 +10,7 @@ import (
 
 func defaultHosts(r region.Region) (hosts []*transport.StatefulHost) {
 	switch r {
-	case region.EU, region.US:
+	case region.DE, region.US:
```

Comment:
This is a breaking change buddy, as people who here using `region.EU` will now be targetting `US`. I think this line should be:

```suggestion
	case region.DE, region.EU, region.US:
```